web/src/components/tasks/cannabis/CannabisPriorityStatusTask.test.tsx

```diff
@@ -362,4 +363,167 @@ describe("<CannabisPriorityStatusTask />", () => {
+    expect(screen.getByText(eligibilityPhrase)).toBeInTheDocument();
+
+    fireEvent.click(screen.getByTestId(randomSocialEquityPriorityType));
+    expect(screen.queryByText(eligibilityPhrase)).not.toBeInTheDocument();
+  });
+
+  it("displays minority/women and impact zone eligibilities when minority/women and impact zone priority types are selected", () => {
+    const randomMinorityOrWomenPriorityType = randomElementFromArray([...priorityTypesObj.minorityOrWomen]);
+    const randomImpactZonePriorityType = randomElementFromArray([...priorityTypesObj.impactZone]);
+
+    const task = generateTask({
+      id: "123",
+      name: "Header",
+    });
+    useMockRoadmapTask(task);
+
+    renderPage(task);
+
+    const eligibilityPhrase = templateEval(Config.cannabisPriorityStatus.phraseWithTwoPriorities, {
+      priorityStatusOne: Config.cannabisPriorityTypes.minorityWomenOrVeteran,
+      priorityStatusTwo: Config.cannabisPriorityTypes.impactZone,
+    });
+
+    expect(screen.queryByText(eligibilityPhrase)).not.toBeInTheDocument();
+
+    fireEvent.click(screen.getByTestId(randomMinorityOrWomenPriorityType));
+    fireEvent.click(screen.getByTestId(randomImpactZonePriorityType));
+    expect(screen.getByText(eligibilityPhrase)).toBeInTheDocument();
+
+    fireEvent.click(screen.getByTestId(randomMinorityOrWomenPriorityType));
+    fireEvent.click(screen.getByTestId(randomImpactZonePriorityType));
+    expect(screen.queryByText(eligibilityPhrase)).not.toBeInTheDocument();
+  });
+
+  it("displays minority/women and social equity eligibilities when minority/women and social equity priority types are selected", () => {
+    const randomMinorityOrWomenPriorityType = randomElementFromArray([...priorityTypesObj.minorityOrWomen]);
+    const randomSocialEquityPriorityType = randomElementFromArray([...priorityTypesObj.socialEquity]);
+
+    const task = generateTask({
+      id: "123",
+      name: "Header",
+    });
+    useMockRoadmapTask(task);
+
+    renderPage(task);
+
+    const eligibilityPhrase = templateEval(Config.cannabisPriorityStatus.phraseWithTwoPriorities, {
+      priorityStatusOne: Config.cannabisPriorityTypes.minorityWomenOrVeteran,
+      priorityStatusTwo: Config.cannabisPriorityTypes.socialEquity,
+    });
+
+    expect(screen.queryByText(eligibilityPhrase)).not.toBeInTheDocument();
+
+    fireEvent.click(screen.getByTestId(randomMinorityOrWomenPriorityType));
+    fireEvent.click(screen.getByTestId(randomSocialEquityPriorityType));
+    expect(screen.getByText(eligibilityPhrase)).toBeInTheDocument();
+
+    fireEvent.click(screen.getByTestId(randomMinorityOrWomenPriorityType));
+    fireEvent.click(screen.getByTestId(randomSocialEquityPriorityType));
+    expect(screen.queryByText(eligibilityPhrase)).not.toBeInTheDocument();
+  });
+
+  it("displays minority/women, impact zone, and social equity eligibilities when minority/women, impact zone, and social equity priority types are selected", () => {
```

Comment:
this is a nitpick, (I'm sorry) but one thing to think about is that tests are documentation for future developers. The names of tests should succinctly communicate the behavior, so that if someone is looking to understand or make changes, they should be able to look at test names and understand and make changes quickly.

These test names are descriptive, but they're fairly wordy and repetitive - different tests here have very similar names. It might be worth considering ways to simplify the name of the test to be clearer about what they're doing. When we were developing together, we were referring to them as "the test for the two-case" and the "test for the three-case". Because that was the Thing they were testing for. A future developer would have to work hard reading the test names & body to see that some are testing the two-case vs three-case.

A few examples that might be more readable. I'm giving more than one to show that this is totally a stylistic nitpick! Your code is just fine, and you're welcome to take any of these suggestions, or none of them. Just communicating that thinking about that "future developer" trying to read & understand the tests is an audience I think about when writing my test names.
- it "displays phrase with 2 eligibilities when minority/women AND impact zone checked"
- it "inserts minority/women and impact zone into 2-part eligibility phrase"
- it "displays 2-part eligibility phrase for minority/women and impact zone"